en/pragma.tex

```diff
@@ -304,6 +304,21 @@ \subsubsection{} In casual conversation the reflexive perfective of
 \NTeri{3/8/2011}{http://naviteri.org/2011/08/new-vocabulary-clothing/}
 
 
+\subsubsection{Slang} At the request of the community, Paul Frommer approved 
+some constructions deviating from standard grammar that could be used as 'slang'
```

Comment:
Txantsan! One more tiny tiny comment: `` `slang'`` instead of `'slang'` (that is LaTeX syntax for the open-quote). Approve otherwise :slightly_smiling_face:

Reply:
Tam, irayo!